lib/skate_web/templates/layout/app.html.eex

```diff
@@ -21,6 +21,7 @@
     <main role="main" class="container">
       <%= render @view_module, @view_template, assigns %>
     </main>
+    <script id="app-user-token" type="text/plain"><%= assigns[:user_token] %></script>
```

Comment:
`<script>` seems like a weird type for this: does this pattern come from somewhere else?

Reply:
I copied this from dotcom, but they use it for complex data and I just have a string. Maybe a `data-user-token` attribute on the `#app` div would be a better approach?

Reply:
Or you could make yours an object:
```elixir
<script id="app-user-token" type="application/json">
  <%= Jason.encode_to_iodata!(%{
    token: assigns[:user_token]
  }) %>
</script>
```